samples/net/nats/src/main.c

```diff
@@ -16,7 +16,7 @@
 /* LED */
 #ifndef LED0_GPIO_CONTROLLER
 #ifdef LED0_GPIO_PORT
-#define LED0_GPIO_CONTROLLER 	LED0_GPIO_PORT
+#define LED0_GPIO_CONTROLLER    LED0_GPIO_PORT
```

Comment:
Unrelated change, please remove this part from this commit.

Reply:
I actually tried to undo all the style related changes. Unfortunately, I was successful in reverting all of them but one-git lint was unhappy with the formatting of the above line (baseline formatting was incorrect)
Is there any way around this? How can I drop all the changes which were made in my initial commit?  Thanks for your help.

Reply:
> How can I drop all the changes which were made in my initial commit?

Given that the "real" changes required for this PR are small, you can just drop this change, and re-do it from scratch. That may as well be the quickest way. Otherwise, the way to deal with these is to add a new commit reverting a particular change (e.g. line) and squash it into previous commit with `git rebase --interactive`. Hope this helps.

Reply:
Yes, it did work. Hopefully, now the changes are good for commit.